src/components/scenes/newAccount/NewAccountReviewScene.tsx

```diff
@@ -89,3 +75,53 @@ const getStyles = cacheStyles((theme: Theme) => ({
     minHeight: theme.rem(3)
   }
 }))
+
+/**
+ * The review scene for new accounts.
+ */
+interface NewAccountReviewProps extends SceneProps<'newAccountReview'> {
+  branding: Branding
+}
+
+export const NewAccountReviewScene = (props: NewAccountReviewProps) => {
+  const { branding, route } = props
+  const dispatch = useDispatch()
+  const { onLogin } = useImports()
+
+  const handleNext = useHandler(() => {
+    logEvent(`Signup_Review_Done`)
+
+    if (onLogin != null) onLogin(route.params.account)
```

Comment:
The old version had an `Airship.clear()` right around here. Was there a reason to drop it?

Reply:
We have a very solid rule in this repo - if you are going to call either `onLogin` or `onComplete`, you must also call `Airship.clear`. We had a lot of bugs where people didn't do this, and random modals would linger. Even if you think it is unneeded, please still do it for safety.